source/type/message.cpp

```diff
@@ -455,7 +455,9 @@ bool Impact::is_relevant(const std::vector<const StopTime*>& stop_times) const {
             }
         }
         return false;
-    } else if (is_rail_section && this->severity->effect == nt::disruption::Effect::REDUCED_SERVICE) {
+    } else if (is_rail_section
+               && (this->severity->effect == nt::disruption::Effect::REDUCED_SERVICE
+                   || this->severity->effect == nt::disruption::Effect::DETOUR)) {
```

Comment:
and this time you didn't use `navitia::contain()` ? :p

Reply:
I decided to start using it at 3, that was arbitrary :grin: